app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::CustomOrganisationsController < AdminController
+  def index
+    @all_orgs = CustomOrganisationName.all
```

Comment:
I know this is within the context of the custom organisation controller but think improving this variable name to something like "custom_organisation_names" could be helpful?